modules/ROOT/pages/fd-deploy-app-from-rm.adoc

```diff
@@ -1,15 +1,18 @@
 = Deploy a Mule Application from Runtime Manager
 
-It might be the case that you are not allowed to deploy an application directly from Flow Designer. Your organization might require someone with the proper authority to deploy applications from Runtime Manager. You can share with that person a copy of your project as a `.jar` file to deploy your application with.
+It might be the case that you are not allowed to deploy an application directly from Flow Designer.
+Your organization might require someone with the proper authority to deploy applications from Runtime Manager.
+You can share with that person a copy of your project as a `.jar` file to deploy your application with.
 
 == Procedure
 
 . In Flow Designer, follow these steps:
 .. Ensure that your project is free of errors.
-.. Click the download icon, which is to the right of the Undo and Redo arrows in the top-left corner of the screen.
-.. Select *Download Package*.
+.. Export your project by clicking the dots next to the Test button and select *Export*.
+.. Select *Export project sources* to export just your project source files, or
+*Export mule application* to export your entire Mule app, then click *Export*.
 +
-*Result:* A `.jar` file that contains a copy of your project is downloaded to the default location on your computer.
+*Result:* A `.jar` file that contains a copy of your project source or Mule app is downloaded to the default location on your computer.
```

Comment:
i'm not sure about this part. If we need to be more specific about the differences between those two, or if the information inside the modal itself is enough. Maybe wait for @mcoaker review

The rest in my opinion is ok :)

Reply:
Fixed.